src/Providers/ModelStubProvider.php

```diff
@@ -47,7 +47,8 @@ public static function generateStubFile(): void
 
         $models_generator_command->run(
             new \Symfony\Component\Console\Input\ArrayInput([
-                '--nowrite' => true
+                '--nowrite' => true,
+                '--reset' => true,
```

Comment:
would you mind explaining what this is fixing/accomplishing please? Maybe an example of a bad docblock that this is fixing?

Reply:
Already included in the MR: https://github.com/psalm/psalm-plugin-laravel/blob/7174fd50e758d899e04220ff6feb5a208a03d0bd/tests/Models/User.php#L13

Reply:
You'll also find an explanation in the commit message, see: 7174fd50e758d899e04220ff6feb5a208a03d0bd